src/logging.h

```diff
@@ -17,7 +17,7 @@
 #include <vector>
 
 static const bool DEFAULT_LOGTIMEMICROS = false;
-static const bool DEFAULT_LOGIPS        = false;
+static const bool DEFAULT_LOGIPS        = true;
```

Comment:
@lbrynaut Is this meant to be here?

Reply:
I'm glad you brought this up. I'm the one that changed the default and checked it in. I found it frustrating during our last hard fork that none of the log files had IP addresses in them. I suppose this is there because some people may be uncomfortable exposing their internal IP addresses. I think that concern is generally a thing of the past, though. I'm certainly open to arguments the other way. I think we should enable IP logging on all instances that we manage.

Reply:
Agreed. If we find it useful we should enable it, but maybe disable it by default for privacy.

Reply:
I'd leave it disabled as well.